grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -401,6 +401,17 @@ class QueryService implements Status {
         results
     }
 
+    void updateCollectionLinkStatus(String consentKey, String projectKey, String status) {
+        final session = sessionFactory.currentSession
+        final String query =
+                ' update consent_collection_link set status = :status ' +
+                'where project_key = :projectKey and consent_key = :consentKey'
+        final sqlQuery = session.createSQLQuery(query)
+        sqlQuery.setParameter('projectKey', projectKey)
+        sqlQuery.setParameter('consentKey', consentKey)
+        sqlQuery.executeUpdate()
+    }
```

Comment:
Would it make more sense to return something useful to the caller? That way any upstream APIs can return the result of the update to the calling page.